test/captcha_signup.test.js

```diff
@@ -33,39 +32,42 @@ describe('captcha on signup', function () {
   describe('svg-captcha', () => {
     describe('when the api returns a new challenge', function () {
       beforeEach(function (done) {
-        this.stub = h.stubGetChallenge([svgCaptchaRequiredResponse1, svgCaptchaRequiredResponse2]);
-        this.lock = h.displayLock('', lockOpts, done);
+        this.stub = h.stubGetChallenge({ required: false });
+        this.stub = h.stubGetSignupChallenge([svgCaptchaRequiredResponse1, svgCaptchaRequiredResponse2]);
+        setTimeout(() =>  {
+          this.lock = h.displayLock('', lockOpts, () =>  {
+            h.clickSignUpTab();
+            h.waitUntilExists(this.lock, '.auth0-lock-with-terms', () => {
+              done();
+            });
+          });
+        }, 1000);
```

Comment:
1000 was the lowest number I was able to set the timeout to and have the tests reliably pass